package.json

```diff
@@ -1,20 +1,21 @@
 {
-  "name": "javascript_playground",
-  "version": "1.0.0",
-  "main": "CreditCardValidation.js",
-  "devDependencies": {
-    "eslint": "^8.56.0",
-    "eslint-config-prettier": "^9.1.0",
-    "prettier": "^3.2.5"
-  },
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "npx eslint '**/*.js'",
-    "format": "npx prettier --write '**/*.js'",
-    "run": "npx http-server"
-  },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
-  "description": ""
+    "name": "javascript_playground",
+    "version": "1.0.0",
+    "main": "CreditCardValidation.js",
+    "devDependencies": {
+        "eslint": "^8.56.0",
+        "eslint-config-prettier": "^9.1.0",
+        "mocha": "^10.3.0",
+        "prettier": "^3.2.5"
+    },
+    "scripts": {
+        "test": "mocha testingWithMocha/tests/factorial_test.js",
```

Comment:
1) I wouldn't recommend the `_test` syntax, and instead change it to `.test` (so `factorial.test.js`). It's not a big deal, but I think introducing underscores into JS file names is a small pet peeve of mine.
2) I'm not sure why it's not picking up/running your test files. I'd have to look into the documentation for mocha, which is something we can do on a phonecall if you'd like

Reply:
Thank You Matt, this is awesome! I had no idea it was possible to use two periods in a file name to be honest 😳
I will try to implement your code (type it myself) and run the tests again. I have a feeling it could have something to do with my assert.throws() function at the end of my tests. Because before I wrote that test case, the suite was working fine.